build-system/tasks/pr-deploy-bot-utils.js

```diff
@@ -23,6 +23,7 @@ const {cyan, green} = require('ansi-colors');
 const {gitCommitHash} = require('../git');
 const {replaceUrls: replaceUrlsAppUtil} = require('../app-utils');
 const {travisBuildNumber} = require('../travis');
+const hostName = `https://storage.googleapis.com/amp-test-website-1/amp_dist_${travisBuildNumber()}`;
```

Comment:
Calling `travisBuildNumber()` in global scope returns an error during all local `gulp` tasks. Sent out a fix in #24021.